slo_generator/backends/cloud_monitoring_mql.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 2019 Google Inc.
```

Comment:
At some point last year, we discussed augmenting the existing Cloud Monitoring backend in order to support MQL. However, I found that MQL uses a different client and different methods. I did not want to make the code unreadable by wrapping all the business logic in `if flag == MQF:`, `elif flag == MQL:`, `elif flag == PromQL`, `else ...` blocks in every method of the class. I ended up defining a new backend entirely. What do you think? The biggest drawback is probably the amount of duplicate code shared by both backends. We could centralize it somewhere else, for example in a separate library.